src/app/(BottomTabNavigation)/AllCases/utils.ts

```diff
@@ -36,8 +36,23 @@ export async function fetchAllCases(userUid: UserUid): Promise<Case[]> {
  * @returns readable date string
  */
 export function formatDate(dateObject: Date) {
-  const date = new Date(dateObject).toDateString().split(' ');
-  return `${date[1]} ${date[2]}, ${date[3]}`;
+  const date = new Date(dateObject);
+  const months = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+  ];
+  const monthName = months[date.getUTCMonth()];
+  return `${monthName} ${date.getUTCDate()}, ${date.getUTCFullYear()}`;
```

Comment:
I'm not sure if this will fix the issue. I think this is a deeper issue than we think because we also have to consider different timezones, daylight savings etc. Look into this [article here](https://javascript.plainenglish.io/react-native-getting-user-device-timezone-and-converting-utc-time-stamps-with-offset-697902d7803) and see what you can find. Assume all dates in our database are stored in UTC time, and we must convert them to the user's specific timezone.

Reply:
I have the code using the moment stuff from the article already so if I should change from the built in Intl. DateTimeFormat object, I can!